daisy/README.md

```diff
@@ -1,4 +1,193 @@
-## Daisy
+# Daisy
 
 Daisy is a GCE workflow tool for running image type workflows on GCE. Development is ongoing.
+https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy
 
+Daisy takes a JSON workflow specification as input. This workflow contains the step information
+for the various steps of the workflow as well as a directed acyclic graph (DAG) of workflow steps.
+Each step may specify dependencies on other steps’ completion, but these dependencies must form a
```

Comment:
This is good information but details. The high level- this is what Daisy does should be spelled out in the simplest terms possible here.